client/src/tests/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from '../App';
+
+test('renders learn react link', () => {
+  render(<App />);
```

Comment:
Running `$ npm test` produces warnings and errors for me and this test fails. Is this expected?